### DISCUSSION TITLE
New app : rhasspy

### DISCUSSION THREAD

**Discussion by jbperrin88** [tteck/Proxmox#1015] [Ideas]
Hi there,

First of all ... thanks so much for all your idea and script ! It's marvelous !! 

I've been using Rhasspy for a long time now i think Master server could be great as a LXC container.

I'll be more and more popular with the adoption of this solution with HomeAssistant .

https://rhasspy.readthedocs.io/en/latest/installation/#docker

Best Regards !

**Comment by cholojuanito** [comment]
I've started working on something. Hopefully I can figure it out since I'm newer to LXC containers but this project seems well put together and generally  easy to put a new container together
